drivers/hmis_external_apis/app/models/hmis_external_apis/mci_mapping.rb

```diff
@@ -0,0 +1,103 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
```

Comment:
Pretty sure the rest of the app has been updated to 2023, not sure why these were missed.